parsl/tests/configs/cori.py

```diff
@@ -0,0 +1,40 @@
+from parsl.config import Config
```

Comment:
There is already a cori_htex.py configuration in this directory. That's a mess. Get rid of one of them.